Resources/Prototypes/Magic/projectile_spells.yml

```diff
@@ -29,9 +29,10 @@
   name: Fireball II
   description: Fires a fireball, but faster!
   components:
+  - type: LimitedCharges
+  - type: AutoRecharge
+    rechargeDuration: 10
   - type: WorldTargetAction
-    useDelay: 10
-    renewCharges: true
```

Comment:
I forgot to remove renew charges from here. The use delay should still be intact.

From discord, @ActiveMammmoth said that "also when used for the first time, the sprite becomes greyed out forever, even after cooldown finishes"